src/graph_notebook/magics/graph_magic.py

```diff
@@ -686,18 +710,22 @@ def on_button_clicked(b):
             logger.info(f'using source_exp: {source_exp}')
             try:
                 load_result = do_load(host, port, source_format.value, ssl, str(source_exp), region_box.value,
-                                      arn.value,
+                                      arn.value, mode.value,
                                       fail_on_error.value, parallelism.value, update_single_cardinality.value,
+                                      queue_request.value, dependencies.value,
```

Comment:
So with the change above, we can then use dependencies.value.split('\n') and we should have a list to feed into our `do_load` function

Reply:
Done